source/helpers/get-rgh-repo-issue-link.ts

```diff
@@ -0,0 +1,3 @@
+export default function getRghRepoIssueLink(issueId: number | string): string {
```

Comment:
Maybe rename this to `getRghRepoIssueUrl` to avoid confusion? I used (nearly) the same function name in https://github.com/sindresorhus/refined-github/pull/4387/files#diff-32553a6819bdc0a556cfcf616e0677aa6ddf6aca02bf4c6b302f3f6e9d6a72a5R27 and that function returns an `HTMLAnchorElement` instead.

Reply:
shouldnt yours be called `createRGHIssueLink`?

Reply:
Indeed. But making a clear distinction between *link* and *URL* is a good idea anyway IMO.